src/telliot_feeds/queries/price/twap.py

```diff
@@ -0,0 +1,75 @@
+""" :mod:`telliot_feeds.queries.price_query`
+
+"""
+import logging
+from dataclasses import dataclass
+from typing import List
+from typing import Tuple
+
+from telliot_feeds.dtypes.float_type import UnsignedFloatType
+from telliot_feeds.dtypes.value_type import ValueType
+from telliot_feeds.queries.abi_query import AbiQuery
+from telliot_feeds.queries.price.spot_price import CURRENCIES
+from telliot_feeds.queries.price.spot_price import SPOT_PRICE_PAIRS
+
+
+logger = logging.getLogger(__name__)
+
+
+def format_spot_price_pairs() -> List[Tuple[str, str]]:
```

Comment:
Is this a repeat of a function created elsewhere that instead can just be imported?

Reply:
Indeed!